src/internal/components/InternalEditor.tsx

```diff
@@ -172,11 +172,13 @@ export const InternalEditor = ({
               refreshPermissions();
             }, [themeModeActive]);
 
+            const { history } = usePuck();
```

Comment:
There's already a usePuck a few lines above this

Reply:
Fixed